rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsVisitor.java

```diff
@@ -69,7 +69,9 @@ public J visit(@Nullable Tree tree, P p, Cursor parent) {
         }
         Iterator<Object> itr = parent.getPath(J.class::isInstance);
         J next = (itr.hasNext()) ? (J) itr.next() : null;
-        preVisit(next, p);
+//        if (next != null) {
+            preVisit(next, p);
+//        }
```

Comment:
```suggestion
        preVisit(next, p);
```

Reply:
@sambsnyd Thanks for spotting. The commented out lines should actually have been the fix. While testing a few more cases I temporarily reverted the fix and in the end accidentally committed it that way...